src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+  background-color: #fff;
+  color: #000;
+
+  &__img {
+    object-fit: contain;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
it's a default value for this style
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/6a378e7f-6715-4f2b-8279-6e00e01ba965)
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/4a9a0030-c229-4dc9-98cb-61023b2a5b0c)

```suggestion
```